tests/testthat/test-lint_file.R

```diff
@@ -109,3 +109,16 @@ test_that("lint() results do not depend on the position of the .lintr", {
     )
   )
 })
+
+test_that("lint uses linter names", {
+  expect_lint("a = 2", list(linter = "bla"), linters = list(bla = assignment_linter), parse_settings = FALSE)
```

Comment:
either here in the test or in the examples (I guess that's waiting on #660 though?), it would be good to present an example like you mentioned, e.g. we could use `linters = list(bla = object_name_linter("SNAKE_CASE"))`

Reply:
The examples of ?exclude sound like a good place for this. Maybe even a vignette? Waiting for #660 since that's the bigger part of the feature.